apps/social/src/App.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+import { useApi, useBalanceSubscription, useLoggedInAccount } from '@gear-js/react-hooks';
+import { Routing } from 'pages';
+import { Header, Footer, ApiLoader } from 'components';
+import { withProviders } from 'hocs';
+import { LOCAL_STORAGE } from './consts';
+import 'App.scss';
+
+function Component() {
+  const { isApiReady, api } = useApi();
+
+  // Set Genesis block
+  useEffect(() => {
+    if (isApiReady) {
+      localStorage.setItem(LOCAL_STORAGE.GENESIS, api.genesisHash.toHex());
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isApiReady]);
+
+  useBalanceSubscription();
+  useLoggedInAccount();
+
+  return (
+    <>
+      <Header />
+      <main>{isApiReady ? <Routing /> : <ApiLoader />}</main>
```

Comment:
Update @gear-je/react-hooks and add   ``const { isLoginReady } = useLoggedInAccount();``
Then add ``const isInitLoading = isApiReady & isLoginReady;`` 
Then ``{isInitLoading ? <Routing /> : <ApiLoader />``

Reply:
If it is possible to get an account, then the account will always be non-null below in the component tree

Reply:
done